shell-pipelines/sort-uniq-head-tail/script-06.sh

```diff
@@ -5,3 +5,4 @@ set -euo pipefail
 # The input for this script is the events.txt file.
 # TODO: Write a command to show how many times anyone has entered and exited.
 # It should be clear from your script's output that there have been 5 Entry events and 4 Exit events.
+sort events.txt | uniq -c
```

Comment:
I'm not sure this quite answers the question - the prompt is that it should be clear that there are 5 Entry events and 4 Exit events, but the output of this is:

```
   2 Entry German
   1 Entry Mariana
   2 Entry Sally
   1 Exit German
   1 Exit Mariana
   2 Exit Sally
```

I can add those numbers up myself to get to 5 and 4, but ideally the script you produce would do that for me?